src/main/java/io/strimzi/kafka/bridge/http/HttpSourceBridgeEndpoint.java

```diff
@@ -138,7 +138,7 @@ private JsonObject buildOffsets(List<HttpBridgeResult<?>> results) {
     }
 
     private int getCodeFromMsg(String msg) {
```

Comment:
There's nothing stopping this method from taking `Exception` as parameter. That way you can say `e instanceof TimeoutException && e.getMessage() != null && e.getMessage().contains("not present in metadata")`, which is less likely to catch a false positive.

Reply:
good point! done ;)